src/python-common/ceph/deployment/drive_group.py

```diff
@@ -241,7 +241,8 @@ def validate(self):
         # type: () -> None
         super(DriveGroupSpec, self).validate()
 
-        if not isinstance(self.placement.host_pattern, six.string_types):
+        if not isinstance(self.placement.host_pattern, six.string_types) and \
+                self.placement.host_pattern is not None:
```

Comment:
Yes, we also need to adapt

https://github.com/ceph/ceph/blob/fee0cda09237ea6d8d9c8cfec31cd3f3555d7dfa/src/pybind/mgr/cephadm/module.py#L2102

Reply:
ack